src/main/java/frc/robot/RobotContainer.java

```diff
@@ -23,18 +23,50 @@ public class RobotContainer {
 	// Replace with CommandPS4Controller or CommandJoystick if needed
 	private final CommandXboxController driverController = new CommandXboxController(
 			OperatorConstants.DRIVER_CONTROLLER_PORT);
+	private final CommandXboxController XboxController = new CommandXboxController.getHID();
```

Comment:
should be `private final XboxController *variable name* = driverController.getHID();`

1. You have already made a CommandXboxController object in the line above as the driverController object
2. You should use the existing object instead of trying to make a new CommandXboxController Class and getting the HID from there
3. The type of `.getHID()` returns XboxController (https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/button/CommandXboxController.html), and you should create a unique variable name for it